runtimes/bun-1.0/src/server.ts

```diff
@@ -38,6 +38,11 @@ const action = async (logger: Logger, request: any) => {
     headers[header.toLowerCase()] = request.headers.get(header);
   });
 
+  const enforcedHeaders = JSON.parse(Bun.env["OPEN_RUNTIMES_HEADERS"] ? Bun.env["OPEN_RUNTIMES_HEADERS"] : '{}');
```

Comment:
It doesnt, because if you do `VAR=` (leaving it empty), it will be empty string. Emtry string is a string so `??` wont do anything. Then it goes to JSON.parse, and empty string cannot be parsed - throwing error as result.